From fda487c0b6a33ffbedf413d27734e37dc36390d4 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Mon, 19 Jun 2017 17:30:12 +0800
Subject: [PATCH 1951/5242] MLK-15119 PCI: imx: remove the wrong rebased codes

commit  02e3660a047a6d96f771fe08a54324be6f0e21dc from
https://source.codeaurora.org/external/imx/linux-imx.git

Some codes are wrong rebase back into mainline in
one git pull rebase confliction.
Remove them.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pci/controller/dwc/pci-imx6.c |    7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/pci/controller/dwc/pci-imx6.c b/drivers/pci/controller/dwc/pci-imx6.c
index 58e88ee..bcd830d 100644
--- a/drivers/pci/controller/dwc/pci-imx6.c
+++ b/drivers/pci/controller/dwc/pci-imx6.c
@@ -1552,13 +1552,6 @@ static int __init imx6_pcie_probe(struct platform_device *pdev)
 	if (of_property_read_u32(node, "cpu-base-addr", &imx6_pcie->cpu_base))
 		imx6_pcie->cpu_base = 0;
 
-	if (IS_ENABLED(CONFIG_EP_MODE_IN_EP_RC_SYS)) {
-		/* add attributes for device */
-		ret = sysfs_create_group(&pdev->dev.kobj, &imx_pcie_attrgroup);
-		if (ret)
-			return -EINVAL;
-	}
-
 	np = of_find_compatible_node(NULL, NULL, "fsl,imx-pcie-phy");
 	if (np != NULL) {
 		imx6_pcie->phy_base = of_iomap(np, 0);
-- 
1.7.9.5

